toolchain/sem_ir/inst_namer.cpp

```diff
@@ -450,6 +450,14 @@ auto InstNamer::CollectNamesInBlock(ScopeId scope_id,
         CollectNamesInBlock(scope_id, inst.decl_block_id);
         break;
       }
+      case CARBON_KIND(ImportDecl inst): {
+        if (inst.package_id.is_valid()) {
+          add_inst_name_id(inst.package_id, ".import");
+        } else {
+          add_inst_name("<default>.import");
```

Comment:
```suggestion
          add_inst_name("default.import");
```
We generally don't use `<...>` as part of valid names (eg, after a `%`), only at the top level for error cases (and for builtin types).

Reply:
Done